dev-lua/luv/luv-1.30.1.1.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils unpacker
+
+# e.g. PV = a.b.c.d
+# a.b.c
+MY_BASE="${PV%.*}"
+# d
+MY_PATCH="${PV##*.}"
+# a.b.c-d
+MY_PV="${MY_BASE}-${MY_PATCH}"
+MY_P="${PN}-${MY_PV}"
+
+DESCRIPTION="Bare libuv bindings for lua"
+HOMEPAGE="https://github.com/luvit/luv"
+# XXX: Remember to check this hash between bumps!
+# https://github.com/luvit/luv/tree/master/deps
+LUA_COMPAT_HASH="daebe77a2f498817713df37f0bb316db1d82222f"
+SRC_URI="
+	https://github.com/luvit/${PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/keplerproject/lua-compat-5.3/archive/${LUA_COMPAT_HASH}.tar.gz -> ${PN}-lua-compat-${PV}.tar.gz
```

Comment:
Check the manifest hashes for these two. They are the same tarball with different name, essentially we are hosting (or will end up hosting) the same file multiple times in all of the mirros. Let's rename it to have a `0.7_p20190919` or similar date-based versioning scheme and continue forward with that until a new release is made.

Reply:
I pinned it to the commit listed in the submodule; should I use the latest release instead?

FWIW the reason this lua-compat thing is being bundled as opposed to me packaging it is that it's source-only and meant to be embedded, as I understand it.

Reply:
I replaced it with pinning to the last release, everything works well.